locales/nb.schema.json

```diff
@@ -2433,7 +2436,7 @@
             "text_style": {
               "label": "Tekststil",
               "options__1": {
-                "label": "Undertekst"
+                "label": "Subtitle"
```

Comment:
Still have some instances of strings being changed to english when they shouldn't